esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -432,6 +433,9 @@ define([
                 this.refreshActionState();
             } else if (name === "hasCompleted") {
                 this.checkIfComplete();
+            } else if (name === "Scope" && newValue) {
+                domClass.remove("scopeOptional", "hidden");
+                domClass.add("scopeOptional", "show");
```

Comment:
The scope text field and label show up in both instances this eliminates one of them.